src/server/object_services/object_server.js

```diff
@@ -1969,9 +1974,6 @@ async function _complete_object_parts(obj) {
 
     const parts = await MDStore.instance().find_all_parts_of_object(obj);
     _complete_next_parts(parts, context);
-    if (context.parts_updates.length) {
-        await MDStore.instance().update_parts_in_bulk(context.parts_updates);
-    }
```

Comment:
To make this a bit safer, Instead of completely removing this call, I would leave it here if there are part_updates, but instead we should change _complete_next_parts and only insert the part_update if the part is uncommitted or has a seq change.

Reply:
Ok. I think I got it, but please review.